Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -667,42 +656,8 @@ void Wiimote::GetExtData(u8* const data)
   // i think it should be unencrpyted in the register, encrypted when read.
   memcpy(m_reg_ext.controller_data, data, sizeof(wm_nc));  // TODO: Should it be nc specific?
 
-  // motionplus pass-through modes
-  if (m_motion_plus_active)
-  {
-    switch (m_reg_motion_plus.ext_identifier[0x4])
-    {
-    // nunchuk pass-through mode
-    // Bit 7 of byte 5 is moved to bit 6 of byte 5, overwriting it
-    // Bit 0 of byte 4 is moved to bit 7 of byte 5
-    // Bit 3 of byte 5 is moved to bit 4 of byte 5, overwriting it
-    // Bit 1 of byte 5 is moved to bit 3 of byte 5
-    // Bit 0 of byte 5 is moved to bit 2 of byte 5, overwriting it
-    case 0x5:
-      // data[5] & (1 << 7)
-      // data[4] & (1 << 0)
-      // data[5] & (1 << 3)
-      // data[5] & (1 << 1)
-      // data[5] & (1 << 0)
-      break;
-
-    // classic controller/musical instrument pass-through mode
-    // Bit 0 of Byte 4 is overwritten
-    // Bits 0 and 1 of Byte 5 are moved to bit 0 of Bytes 0 and 1, overwriting
-    case 0x7:
-      // data[4] & (1 << 0)
-      // data[5] & (1 << 0)
-      // data[5] & (1 << 1)
-      break;
-
-    // unknown pass-through mode
-    default:
-      break;
-    }
-
-    ((wm_motionplus_data*)data)->is_mp_data = 0;
-    ((wm_motionplus_data*)data)->extension_connected = m_extension->active_extension;
-  }
+  reinterpret_cast<wm_motionplus_data*>(data)->is_mp_data = 0;
```

Comment:
I missed this previously, but you should probably only do this when m_motion_plus_active is true.